### DISCUSSION TITLE
How do I pre-build a font alias texture and store it locally, so that imgui can use it later?

### DISCUSSION THREAD

**Discussion by Advik-B** [ocornut/imgui#7129] [New Users Build/Link/Run issues ONLY!]
This question stems from https://github.com/ocornut/imgui/discussions/7126

**Comment by ocornut** [comment]
Hello, this isn't the right section to ask. Anyway it is not normally possible, but see ideas like #6169.
I quite dislike this, I believe you should solve your #7126 problem instead.
